torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -224,7 +224,12 @@ void runDiffGraphPasses(std::shared_ptr<Graph>& graph) {
     // TupleConstruct / TupleUnpack pairs can still be present at this point
     // and must be removed for fusion.
     LowerSimpleTuples(graph);
-    GRAPH_DUMP("After LowerSimpleTuples, before BatchMM", graph);
+    GRAPH_DUMP(
```

Comment:
Can you add a comment abt why we're calling `RemoveProfileNodesAndSpecializeTypes`, and that any pass that uses specialized tensor types has to be run after  `FuseTensorExprs`.